swan-lake/development-tutorials/source-code-dependencies/style-guide/expressions.md

```diff
@@ -143,7 +143,12 @@ string[] names = [
 
 ## Type casting
 
-* Do not keep spaces between the type and the angle brackets (i.e., `<string>`).
+* Do not keep spaces between the type and the angle brackets. 
+
+**Example,**
+
+`<string>`
```

Comment:
Shall we add at least a type cast expression here?

````suggestion
```ballerina
<int>id;
```
````

Or we can combine this point and the next and just remove this example.

Reply:
Accepted the suggestion for the example. Let's keep both points as it is clearer.